__tests__/__mock_helpers/wallet-service.fixtures.ts

```diff
@@ -35,6 +35,7 @@ export const buildSuccessTxByIdTokenDataResponse = (): string => {
     txTokens,
   });
 };
+// eslint-disable-next-line -- 'arrow-parens' rule conflicts with typescript syntax here
```

Comment:
[`arrow-parens`](https://eslint.org/docs/latest/rules/arrow-parens) requires the parenthesis to be removed when there is only one parameter.

But since the single parameter is optional in typescript, the resulting code would be something like `const fun = param? => 'result'`, which is invalid syntax for typescript.

However, this rule is useful - especially because it can run automatically in many cases - and I believe it's still good to add it.